src/components/inline/Button/Button.stories.js

```diff
@@ -7,9 +7,41 @@ export default {
   component: Button,
 };
 
-const Template = ({ children }) => <Button>{children}</Button>;
+const Template = ({
+  externalLinkButton,
+  internalLinkButton,
+  scriptActionButton,
+}) => (
+  <div
+    style={{
+      display: "flex",
+      flexDirection: "row",
+      justifyContent: "space-evenly",
+      alignItems: "center",
+      marginTop: "50vh",
+    }}
+  >
+    <Button button={externalLinkButton} />
+    <Button button={internalLinkButton} />
+    <Button button={scriptActionButton} />
```

Comment:
I think you probably want to make these separate Storybook stories, since they are each an example of a different usage of the component.